testng-core/src/test/java/test/listeners/issue2685/InterruptedInListenerTest.java

```diff
@@ -0,0 +1,23 @@
+package test.listeners.issue2685;
+
+import org.assertj.core.api.Assertions;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import org.testng.xml.XmlSuite;
+import test.SimpleBaseTest;
+
+public class InterruptedInListenerTest extends SimpleBaseTest {
+
+  @Test(description = "GITHUB-2685")
+  public void testThreadIsNotInterruptedInListener() {
```

Comment:
We are trying to revamp the tests and ensure that the tests are part of just 1 class. Can you please help move this test into `test.listeners.ListenersTest` so that we can avoid having one more extra class ?

Reply:
Great idea, I was quite confused when I saw all those test classes.